doc/developers/advanced_installation.rst

```diff
@@ -284,58 +298,3 @@ build step::
     python setup.py build --compiler=my_compiler install
 
 where ``my_compiler`` should be one of ``mingw32`` or ``msvc``.
-
-
-.. _testing:
-
-Testing
-=======
```

Comment:
I just saw this part which seems redundant so I removed it. It isn't referenced anywhere.

Ping @adrinjalali 	@amueller 	@jnothman if you're OK with it

Reply:
It's kinda some good info (although I never had read it before :P); but no strong feelings here.

Reply:
No strong opinion either. Conciseness is good.

Reply:
No strong opinion either. Conciseness is good.